sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceStrategy.scala

```diff
@@ -263,6 +265,10 @@ class FindDataSourceTable(sparkSession: SparkSession) extends Rule[LogicalPlan]
     })
   }
 
+  private def format(name: String): String = {
```

Comment:
why do we need this for catalog name only?

Reply:
removed since we do not need to handle catalog specially.